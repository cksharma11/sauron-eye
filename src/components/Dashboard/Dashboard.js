import React from "react";
import DashboardTable from "../DashboardTable/DashboardTable";
import { getSortedDashboard } from "../../helpers/helpers";

const Dashboard = ({ pushers }) => {
  const sortedDashboard = getSortedDashboard(pushers);

  return (
    <div className="dashboard">
      <DashboardTable dashboard={sortedDashboard} />
    </div>
  );
};

export default Dashboard;
