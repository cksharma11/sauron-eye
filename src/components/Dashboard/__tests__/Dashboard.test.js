import React from "react";
import { shallow, configure } from "enzyme";
import mockPushers from "../../../data/mock.json";
import Adapter from "enzyme-adapter-react-16";
import Dashboard from "../Dashboard";

configure({ adapter: new Adapter() });

describe("Dashboard", () => {
  it("should render properly", () => {
    const wrapper = shallow(<Dashboard pushers={mockPushers} />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
