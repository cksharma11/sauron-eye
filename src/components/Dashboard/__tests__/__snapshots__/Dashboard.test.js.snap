// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should render properly 1`] = `
"<div className=\\"dashboard\\">
  <DashboardTable dashboard={{...}} />
</div>"
`;
